libmscore/barline.cpp

```diff
@@ -100,6 +100,11 @@ static void undoChangeBarLineType(BarLine* bl, BarLineType barType, bool allStav
             case BarLineType::END_REPEAT:
                   m->undoChangeProperty(Pid::REPEAT_END, true);
                   break;
+            case BarLineType::END_START_REPEAT:
```

Comment:
I don't think this code should be needed.  We should never actually be changing the barline type to this.